lib/api/v1/incident-controller.js

```diff
@@ -34,14 +40,20 @@ module.exports = function(app, user) {
 
   // Get an Incident by _id
   app.get('/api/v1/incident/:_id', user.can('view data'), function(req, res) {
+    var referer = req.headers.referer ? req.headers.referer : '';
     Incident
       .findById(req.params._id)
       .populate({ path: 'creator', select: 'username' })
       .populate({ path: 'assignedTo', select: 'username' })
       .exec(function(err, incident) {
         if (err) res.send(err.status, err.message);
         else if (!incident) res.send(404);
-        else res.send(200, incident);
+        else {
+          if (!(referer.split('/')[referer.split('/').length - 1] === 'reports')) {
```

Comment:
Would it be better to check `if (referer.split('/')[referer.split('/').length - 3] === '_id')`? Not exactly sure if that works.